lib/android-helpers.js

```diff
@@ -49,6 +49,12 @@ const CHROME_BROWSER_PACKAGE_ACTIVITY = {
 const SETTINGS_HELPER_PKG_ID = 'io.appium.settings';
 const SETTINGS_HELPER_UNLOCK_ACTIVITY = '.Unlock';
 const EMULATOR_PATTERN = /\bemulator\b/i;
+// These constants are in sync with
+// https://developer.apple.com/documentation/xctest/xcuiapplicationstate/xcuiapplicationstaterunningbackground?language=objc
+const APP_STATE_NOT_INSTALLED = 0;
```

Comment:
I think it could be more readable to make an APP_STATE frozen object and put these values into the object

Reply:
Addressed as https://github.com/appium/appium-android-driver/pull/699/commits/030df34a0726e9bf447021da58b5a2cb071741eb